### PR TITLE
convert crio periodics to communinity clusters

### PR DESCRIPTION
**Pull request by kannon92**
#30888 

We are able to convert most of these jobs but I left the ones that do not have corresponding ignition files.

### CODE REVIEW

File: config/jobs/kubernetes/sig-node/crio.yaml [hunk 1]
```diff
@@ -27,16 +28,17 @@ periodics:
       - --deployment=node
       - --env=KUBE_SSH_USER=core
       - --gcp-zone=us-west1-b
-      - --gcp-project-type=node-e2e-project
```
Comment (kannon92) [inline config/jobs/kubernetes/sig-node/crio.yaml:30]: Community cluster does not have these project types.

File: config/jobs/kubernetes/sig-node/crio.yaml [hunk 2]
```diff
@@ -27,16 +28,17 @@ periodics:
...
       - --node-tests=true
       - --provider=gce
       - --test_args=--nodes=8 --focus="\[NodeConformance\]" --skip="\[Flaky\]|\[Slow\]|\[Serial\]"
       - --timeout=180m
-      - --node-args=--image-config-file=/home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/crio/latest/image-config-cgrpv1.yaml
+      - --node-args=--image-config-file=/home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/crio/latest/image-config-cgrpv1-k8s-infra-prow-build.yaml
       env:
       - name: GOPATH
         value: /go
+      - name: IGNITION_INJECT_GCE_SSH_PUBLIC_KEY_FILE
```
Comment (kannon92) [inline config/jobs/kubernetes/sig-node/crio.yaml:40]: This environment variable injects a ssh public key that we specifiy in the ignition file.

File: config/jobs/kubernetes/sig-node/crio.yaml [hunk 3]
```diff
@@ -383,57 +396,6 @@ periodics:
     testgrid-tab-name: ci-crio-cgroupv1-node-e2e-resource-managers
```
Comment (saschagrunert) [inline config/jobs/kubernetes/sig-node/crio.yaml:396]: What about `ci-crio-cgroupv1-node-e2e-resource-managers`?

Comment (kannon92) [reply config/jobs/kubernetes/sig-node/crio.yaml:396]: There is a special ignition file for that one.

Comment (saschagrunert) [reply config/jobs/kubernetes/sig-node/crio.yaml:396]: So no `cluster: k8s-infra-prow-build` for this one?

Comment (kannon92) [reply config/jobs/kubernetes/sig-node/crio.yaml:396]: Yea due to https://github.com/kubernetes/test-infra/blob/master/jobs/e2e_node/crio/latest/image-config-cgrpv1-resource-managers.yaml

**Review by saschagrunert — APPROVED** [review]

**Comment by kannon92** [comment]
/retest

rebase label seems to be off.  I resolved conflicts and have pushed up a resolutoin.

**Review by saschagrunert — APPROVED** [review]

**Comment by saschagrunert** [comment]
PTAL @yujuhong @Random-Liu @dchen1107 @derekwaynecarr @sjenning @mrunalp @klueska @SergeyKanzhelev @endocrimes
for approval

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm
/approve

**Comment by saschagrunert** [comment]
@kubernetes/test-infra-admins may I ask you to manually remove the `needs-rebase` label?

/lgtm

**Comment by saschagrunert** [comment]
Ah the label is gone :)

**Pull request merged**